fineract-avro-schemas/src/main/avro/loan/v1/LoanRepaymentDueDataV1.avsc

```diff
@@ -5,31 +5,23 @@
     "fields": [
         {
             "default": null,
-            "name": "id",
+            "name": "loanId",
             "type": [
                 "null",
                 "long"
             ]
         },
         {
             "default": null,
-            "name": "accountNo",
+            "name": "loanAccountNo",
```

Comment:
I think let's make this not-nullable. (Please double-check if a loan always has accountNo)